mailer.rst

```diff
@@ -127,7 +128,7 @@ Service               Install with
 .. versionadded:: 6.4
 
     The ``Brevo`` integration was renamed in Symfony 6.4 (in previous
-    Symfony versions it was called ``Sendinblue``).
+    Symfony versions it was called ``Sendinblue``) and the Scaleway integration was introduced in Symfony 6.4.
```

Comment:
The ``Brevo`` (in previous Symfony versions it was called ``Sendinblue``) and `Scaleway` integrations were introduced in Symfony 6.4.